### PR TITLE
Backend data models

### PR DESCRIPTION
**Pull request by chadzink**
This is the RESTFULL API for the known backend objects

### CODE REVIEW

**Comment by chadzink** [comment]
@Samsinite can you check the has_many_and_belongs to relationship for event and speakers? I think I did it wrong in the models, serializer, or policy.

**Comment by Samsinite** [comment]
You bet, I'll take a look tonight

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  updated_at :datetime         not null
+#
+# Foreign Keys
+#
+#  fk_rails_c6b77e33aa  (event_id => events.id)
+#
+
+
+class Comment < ActiveRecord::Base
+  belongs_to :event, inverse_of: 'comments'
```
Comment (Samsinite) [inline app/models/comment.rb:19]: belongs_to :author

Comment (chadzink) [inline app/models/comment.rb:19]: Added in seed data PR

**Pull request merged**